### PR TITLE
#860qntvby : return tx object instead of hex

### PR DESCRIPTION
**Pull request by cainnusti**
All functions will return transaction object instead of hexadecimal format with the exception of the create swap offer function as this is not broadcasted but only provided as the transaction to be used in the atomic swap.

https://app.clickup.com/t/860qntvby

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "stas-js",
-  "version": "1.2.4",
+  "version": "1.2.8",
```
Comment (adityapanther-vaionex) [inline package.json:3]: @cainnusti why are we jumping from `1.2.4` to directly `1.2.8`

**Review by adityapanther-vaionex — COMMENTED** [review]
@cainnusti please comment

File: package.json (hunk 1 above)
Comment (cainnusti) [reply package.json:3]: Seems the previous version were not updated in the package.json in stas-js. Current latest version is 1.2.7. Is this still the right approach in this case? @adityapanther-vaionex

Comment (adityapanther-vaionex) [reply package.json:3]: got it @cainnusti 
thanks

**Review by adityapanther-vaionex — APPROVED** [review]
LGTM

**Pull request merged**